libraries/app/application.cpp

```diff
@@ -982,16 +985,18 @@ void application::initialize(const fc::path& data_dir, const boost::program_opti
          try {
             genesis_state = fc::json::from_file(genesis_out).as<genesis_state_type>( 20 );
          } catch(const fc::exception& e) {
-            std::cerr << "Unable to parse existing genesis file:\n" << e.to_string()
-                      << "\nWould you like to replace it? [y/N] ";
+            ilog(
```

Comment:
This prompts for user input, please change back to original.

Reply:
ok sure ! Thanks !